NSS-Day-Cohort-42/rare-API-media-moguls
# Cb category fixtures

#### Changes Made
1. Added file `categories.json` to `fixtures` directory.

​
#### Steps to Review
1. Checkout this branch locally and run the application.
```
git fetch --all
git checkout cb-category-fixtures
```
2. Run the application.
```
npm start
```
3. Open a new Terminal tab (⌘T) and navigate to the server directory.
```
..
python manage.py runserver
```
4. Initialize virtual environment, and run the server.
```
pipenv shell

```
5. Test app functionality.
> should allow user to makemigrations, migrate, and loaddata for categories. should in turn allow for posts and other dependent data sets to migrate and load
6. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
